### PR TITLE
Improve user messages by following a consistent format

### PR DESCRIPTION
**Pull request by pradh**
This was programmed together with Jennifer and Carolyn.

### CODE REVIEW

File: util/src/main/java/org/datacommons/util/ComplexValueParser.java [hunk 1]
```diff
@@ -296,45 +294,44 @@ private boolean parseQuantityRange(String startVal, String endVal, String unit)
       logCtx.addEntry(
           Debug.Log.Level.LEVEL_ERROR,
           "MCF_QuantityRangeMalformedValues",
-          "Malformed start component in QuantityRange value ("
+          "Malformed start component in QuantityRange value; must be a number of '-' :: value: '"
```
Comment (chejennifer) [inline util/src/main/java/org/datacommons/util/ComplexValueParser.java:297]: number or

File: util/src/main/java/org/datacommons/util/ComplexValueParser.java [hunk 2]
```diff
@@ -296,45 +294,44 @@ private boolean parseQuantityRange(String startVal, String endVal, String unit)
...
+              + "'",
           mainNode.getLocationsList());
       return false;
     }
     if (!(McfUtil.isNumber(endVal) || endVal.equals("-"))) {
       logCtx.addEntry(
           Debug.Log.Level.LEVEL_ERROR,
           "MCF_QuantityRangeMalformedValues",
-          "Malformed end component in QuantityRange value ("
+          "Malformed end component in QuantityRange value; must be a number of '-' :: value: '"
```
Comment (chejennifer) [inline util/src/main/java/org/datacommons/util/ComplexValueParser.java:311]: number or

**Review by chejennifer — APPROVED** [review]
nice!

File: util/src/main/java/org/datacommons/util/ComplexValueParser.java (hunk 1 above)
Comment (pradh) [reply util/src/main/java/org/datacommons/util/ComplexValueParser.java]: Done

File: util/src/main/java/org/datacommons/util/ComplexValueParser.java (hunk 2 above)
Comment (pradh) [reply util/src/main/java/org/datacommons/util/ComplexValueParser.java]: Done

**Comment by pradh** [comment]
Thanks for the find!

**Pull request merged**